openquake/calculators/base.py

```diff
@@ -54,6 +54,19 @@ class AssetSiteAssociationError(Exception):
 rlz_dt = numpy.dtype([('uid', (bytes, 200)), ('weight', float)])
 
 
+def set_array(longarray, shortarray):
+    """
+    :param longarray: an array of length L >= l
+    :param shortarray: an array of length l
```

Comment:
Please specify in the docstring that those should be numpy arrays. For lists, for instance, it breaks.